manifests/s/StandardNotes/StandardNotes/3.9.20/StandardNotes.StandardNotes.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: StandardNotes.StandardNotes
+PackageVersion: 3.9.20
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://github.com/standardnotes/desktop/releases/download/v3.9.20/standard-notes-3.9.20-win-ia32.exe
```

Wrong url, my next push might fix it tho
```suggestion
  InstallerUrl: https://github.com/standardnotes/desktop/releases/download/v3.9.20/standard-notes-3.9.20-win.exe
```